ansible/dump_inventory.py

```diff
@@ -32,5 +33,6 @@ def natural_sort_key(s, _nsre=re.compile('([0-9]+)')):
         host_ip = host
         if 'ansible_host' in hostvars[host]:
             host_ip = hostvars[host]['ansible_host']
-        print("- %s (%s) [status](http://%s:3013/v2/status) [top](http://%s:3013/v2/blocks/top)" % (host, host_ip, host_ip, host_ip))
+        bullet = '*' if 'kind' in hostvars[host]['tags'] and hostvars[host]['tags']['kind'] == 'seed' else '-'
```

Comment:
How this render in markdown ? The output is and should be kept as valid markdown. Once rendered, users should be able to differentiate between the two type of node.

Alternative implementation would be subgroups: seeds, peers, this implementation is good tho, given the above concerns are not valid.

Reply:
Sorry, never thought output actually should be in some valid format, will change it